sssom/util.py

```diff
@@ -231,6 +233,8 @@ def filter_redundant_rows(
     else:
         key = [SUBJECT_ID, OBJECT_ID, PREDICATE_ID]
     dfmax: pd.DataFrame
+    df[CONFIDENCE] = df[CONFIDENCE].replace("", np.nan)
```

Comment:
I feel this should be more principled.. We already say in LinkML that something should be interpreted as a double, so we need to retain that typing information. We don't want to add lines like this for every single numeric veriable.

Reply:
Done!